addons/website_event/views/event_templates_list.xml

```diff
@@ -350,11 +350,8 @@
                             </small>
                         </div>
                         <!-- Location -->
-                        <t t-set="has_city_or_country" t-value="event.address_id.sudo().city or event.address_id.sudo().country_id"/>
-                        <div t-if="not event.address_id or has_city_or_country" class="d-flex align-items-center">
-                            <i class="fa fa-map-marker me-2" title="Location"/>
-                            <small t-if="event.address_id" class="o_not_editable fw-bold" itemprop="location" t-out="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'no_marker': 'true'}"/>
-                            <small t-else="" class="o_not_editable fw-bold" itemprop="location">Online event</small>
+                        <div class="d-flex align-items-center">
+                            <small class="o_not_editable fw-bold" itemprop="location" t-out="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'null_text': 'Online event'}"/>
```

Comment:
I can see the idea there but it's not really what I had in mind, I was thinking more something like this below. I don't think we need to move the `has_city_or_country` logic from the event module as it's a bit too specific imo.

```suggestion
                        <t t-set="has_city_or_country" t-value="event.address_id.sudo().city or event.address_id.sudo().country_id"/>
                        <div t-if="not event.address_id or has_city_or_country" class="d-flex align-items-center">
                            <small t-if="event.address_id" class="o_not_editable fw-bold" itemprop="location" t-out="event.address_id"
                                t-options="{'widget': 'contact', 'fields': ['city'], 'null_text': 'Online event'}"/>
```
And good catch as we can also remove the custom map marker as it can also be manage with the widget directly :+1: